runtime/src/genesis_utils.rs

```diff
@@ -307,6 +312,15 @@ pub fn create_genesis_config_with_leader_ex_no_features(
         native_mint_account,
     ));
 
+    let epoch_rewards_account = AccountSharedData::create(
+        1,
+        vec![0; EpochRewards::size_of()],
+        sysvar::id(),
+        false,
+        u64::MAX,
+    );
+    initial_accounts.push((epoch_rewards::id(), epoch_rewards_account));
```

Comment:
based on watching the account on `solana-test-validator` there is no reason to get clever with the lamports here because it gets reset to the rent-exempt minimum at the first epoch boundary anyway. we could hardcode the value `1454640` but i avoided getting it via `rent.minimum_balance()` because `lamports_per_signature` is `0` in some tests, so im wary using `Rent` during genesis is brittle

Reply:
We're using rent during genesis when creating the stake config: https://github.com/anza-xyz/agave/blob/b88655cb60f31f08fed1669b78c68e27b3408e41/programs/stake/src/config.rs#L46-L50

So let's do that here as well.

Also, is there a reason to add the new account here as opposed to here? https://github.com/anza-xyz/agave/blob/b88655cb60f31f08fed1669b78c68e27b3408e41/programs/stake/src/lib.rs#L16-L18

That way, totally new Solana networks using `solana-genesis` will also get the account: https://github.com/anza-xyz/agave/blob/b88655cb60f31f08fed1669b78c68e27b3408e41/genesis/src/main.rs#L750